handbook/engineering/distribution/goals.md

```diff
@@ -2,78 +2,91 @@
 
 Goals are continuously updated and reviewed. If you find these goals do not reflect our current priorities or are out of date, please update them as soon as possible or add it as a topic to our [weekly sync](recurring_processes.md#weekly-distribution-team-sync).
 
-## Active goals
+## Goals
 
 Progress toward our active goals is described in our [tracking issue](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aopen+is%3Aissue+label%3Atracking+label%3Ateam%2Fdistribution).
 
-### [Upgrades between releases are easy to perform](https://github.com/orgs/sourcegraph/projects/71)
+### [Collect metrics and feedback about Sourcegraph deployments](https://github.com/orgs/sourcegraph/projects/143)
 
-Performing upgrades to deployments is currently a complicated process that requires keeping a fork of our configuration and resolving diff conflicts when performing upgrades which are often complicated as the configuration might contain environment-specific customization. This process creates a bad experience for our customers because of the unknown amount of effort of the upgrade process.
-We will start by looking at our Kubernetes deployment and working on an easier update process.
+Sourcegraph collects a number of data points through pings that are used to understand how customers use Sourcegraph and drive how we improve the product. We currently do not have any metrics, pings or manual data points collected from site-admins, which makes it impossible for the Distribution team to make educated guesses on features that site-admins require and how projects should be prioritized.
 
-- **Owner**: Geoffrey, Uwe
-- **Status**: In progress. Estimated completion FY21-Q1.
+- **Status**: In Progress. _Estimate FY22-Q1_
 - **Outcomes**:
-  - Upgrades to deployments do not require resolving diff conflicts from upstream.
-  - Upgrading a deployment configuration requires less than 2 hours of work.
+  - We have a tool in place to store and query site-admin data.
```

Comment:
I'm already feeling this pain point, I hesitant to collect customer feedback because I don't know where to put it.